### DISCUSSION TITLE
How to Render On this page section on the right side of the screen.

### DISCUSSION THREAD

**Discussion by Its-suLav-D** [quarto-dev/quarto-cli#3926] [Q&A] [answered]
Hey Y'all,

I am a beginner in Quarto and I have a super quick question. Right Now My Page looks something like this:

![Screenshot 2023-01-10 at 2 55 22 PM](https://user-images.githubusercontent.com/63928979/211670653-cb6c9125-2db6-4887-bf63-37de6e2723b9.png)

However, I want to have something like this 
![Screenshot 2023-01-10 at 2 57 00 PM](https://user-images.githubusercontent.com/63928979/211670890-0b9d77ab-3032-4077-aa31-1d08ae754726.png)

Basically, Move the "On this page" section to the right side of the screen to make it look good like the above image.

As I was adding Headings the contents inside of the "On this page" were generated automatically. So, I am wondering if there is any easy fix to that. I appreciate your response. Thank.

**Comment by Its-suLav-D** [comment, marked as answer]
Never Mind I figured it out. It was toc location.
